benchmarking/definitions/definition_nashpobench.py

```diff
@@ -24,6 +26,21 @@
 ]
 
 
+_config_space = {
```

Comment:
Could you add a comment there? What the boolean does may not be obvious for a reader who dont have all the context.